### ISSUE TITLE
Bug: Boats on Quicksoil/Quicksoilglass

### ISSUE THREAD

**Issue by Lollywa** [devs-immortal/The-Aether#550]
### What happened?

If you give a boat momentum on Quicksoil or Quicksoilglass it accelerates to hyper speed. When I sat in it I reached up to 400 blocks per second before I stoppen on a chunkborder because my game couldn't keep up with loading new chunks.

### Version

1.6.1 (Latest)

### Fabric API Version

0.43.1

### Trinkets Version

3.0.4

### Other mods

```shell
modmenu 2.0.4
carpetmod 1.4.45+v210811 and carpet-extra 1.4.43
malilib 0.10.0
minihud 0.19.0
tweakeroo 0.10.0
starlight 1.0.0-RC3
iris-mc 1.1.1
```

**Comment by Jack-Papel** [comment]
Resolved, expect this in 1.6.2

**Comment by Jack-Papel** [comment]
Closed as fixed in 1.6.1.2
